bsef.csl

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="fr-FR" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Bulletin de la Société Entomologique de France</title>
+    <title-short>bsef</title-short>
+    <id>http://www.zotero.org/styles/bsef</id>
+    <link href="http://www.zotero.org/styles/bsef" rel="self"/>
+    <link href="http://www.lasef.org/publications/le-bulletin-de-la-sef/" rel="documentation"/>
+    <author>
+      <name>Quentin Rome</name>
+      <email>rome@mnhn.fr</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="zoology"/>
+    <category field="biology"/>
+    <issn>0037-928X</issn>
+    <updated>2016-11-11T17:22:46+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="and">et</term>
+      <term name="in">in</term>
+      <term name="from">de</term>
+      <term name="retrieved">récupéré</term>
+      <term name="accessed">consulté le</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" font-variant="small-caps" suffix=".">
+      <name font-variant="normal" suffix="." and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="15" initialize-with=". " name-as-sort-order="all" sort-separator=" ">
+        <name-part name="family" text-case="capitalize-first" font-variant="small-caps"/>
+      </name>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" suffix=", editor(s). ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always">
+        <name-part name="family"/>
```

Comment:
the name part here doesn't do anything. Just delete these two lines (39 and 40) and put an endings slash at the end of line 38.